### ISSUE TITLE
Multiple Object reference not set to an instance of an object

### ISSUE THREAD

**Issue by danbudworthmead** [Impostor/Impostor#411]
# Bug Report

## Base Information
- Operating System: ubuntu
- Impostor Version: latest code built locally
- Among Us Version: 2021.5.10/12

## I confirm:
- [x ] that I have searched for an existing bug report for this issue.

![image](https://user-images.githubusercontent.com/9319945/118202072-fc682880-b450-11eb-9a1d-99abf2bbc9b1.png)

**Comment by JohnTranQUT** [comment]
**Interesting.**
Can you provite more information?
It's because I have pulled the latest version, compiled and did not run into this error on the Ubuntu servers I have it reside on.
Does this happen upon connection/spawning?
Once the game has started?

**Comment by Mr-VIT** [comment]
@danbudworthmead try to disable all plugins, if it helps, then rebuild the plugins with latest API

**Comment by danbudworthmead** [comment]
@JohnTranQUT Yeah repro steps are; have thousands of people joining and leaving over the course of 24 hours.

@Mr-VIT can't really do that as this is skeld.net production server.

**Comment by danbudworthmead** [comment]
After doing some more testing on the skeld.net production server I've realised that sometimes PlayerInfo can be null.
![image](https://user-images.githubusercontent.com/9319945/118362918-c11d4500-b589-11eb-904f-065402334481.png)
This hits the catch and so no longer prints out the error. But there's a bigger issue here as PlayerInfo is sometimes null.

**Comment by realfaceitkantesilvestrs** [comment]
This would be a great time to have a tool that would spawn thousands of fake players to stress test

**Comment by danbudworthmead** [comment]
Yeah. I just stress test on the production server haha.
